src/views/AddUnitsZeroStateView/__stories__/AddUnitsViewZeroState.stories.tsx

```diff
@@ -23,10 +23,10 @@ const Template: ComponentStory<typeof AddUnitsView> = function () {
         backgroundSize: 'cover',
       }}
     >
-      <AddUnitsView />
+      <AddUnitsView {...args} />
     </Grid>
   );
 };
 
 export const Default = Template.bind({});
-Default.args = {};
+Default.args = { faction: 'BRETTONIA' };
```

Comment:
I think that this should correspond to an exact ID for brettonia. Our lookups will be case sensitive